src/course_bot/csa.clj

```diff
@@ -69,23 +69,29 @@
       (pres/drop-talk db conf "lab1" true)
       (pres/all-scheduled-descriptions-dump-talk db conf "lab1")
 
-;      (essay/submit-talk db conf "essay1")
-;      (essay/status-talk db conf "essay1")
-;      (essay/assignreviewers-talk db conf "essay1")
-;      (essay/review-talk db conf "essay1")
-;      (essay/myfeedback-talk db conf "essay1")
+      (if (contains? conf :essay1)
+        (do
+          (essay/submit-talk db conf "essay1")
+          (essay/status-talk db conf "essay1")
+          (essay/assignreviewers-talk db conf "essay1")
+          (essay/review-talk db conf "essay1")
+          (essay/myfeedback-talk db conf "essay1")))
 
-;      (essay/submit-talk db conf "essay2")
-;      (essay/status-talk db conf "essay2")
-;      (essay/assignreviewers-talk db conf "essay2")
-;      (essay/review-talk db conf "essay2")
-;      (essay/myfeedback-talk db conf "essay2")
+      (if (contains? conf :essay2)
+       (do
+         (essay/submit-talk db conf "essay2")
+         (essay/status-talk db conf "essay2")
+         (essay/assignreviewers-talk db conf "essay2")
+         (essay/review-talk db conf "essay2")
+         (essay/myfeedback-talk db conf "essay2")))
 
-;      (essay/submit-talk db conf "essay3")
-;      (essay/status-talk db conf "essay3")
-;      (essay/assignreviewers-talk db conf "essay3")
-;      (essay/review-talk db conf "essay3")
-;      (essay/myfeedback-talk db conf "essay3")
+      (if (contains? conf :essay3)
+       (do
+         (essay/submit-talk db conf "essay3")
+         (essay/status-talk db conf "essay3")
+         (essay/assignreviewers-talk db conf "essay3")
+         (essay/review-talk db conf "essay3")
+         (essay/myfeedback-talk db conf "essay3")))
```

Comment:
- When is much better.
- Do you actually check it? I don't sure hat `nil` value is available in `handlers/defhandler`?